### ISSUE TITLE
Website image analysis

### ISSUE THREAD

**Issue by anna-zhang** [anna-zhang/image-text#2]
- [x] get an array of all image links
- [x] split array by those with bad alt text and those with no alt text
- [ ] don't get logos (role="logo", type="logo")
- [x] pass through image analysis to generate labels
